tobaccocessation/main/models.py

```diff
@@ -11,18 +10,17 @@
     HISPANIC_LATINO_CHOICES
 
 
-@python_2_unicode_compatible
 class UserProfile(models.Model):
     #  ALL_CU group affiliations
     user = models.OneToOneField(User, related_name="profile", unique=True,
                                 on_delete=models.CASCADE)
-    gender = models.CharField(max_length=1, choices=GENDER_CHOICES)
-    is_faculty = models.CharField(max_length=2, choices=FACULTY_CHOICES)
-    institute = models.CharField(max_length=2, choices=INSTITUTION_CHOICES)
-    specialty = models.CharField(max_length=3, choices=SPECIALTY_CHOICES)
-    hispanic_latino = models.CharField(max_length=1,
+    gender = models.CharField(max_length=5, choices=GENDER_CHOICES)
```

Comment:
Why did the lengths need to change for these fields? If needed, we'll also need a data migration for this update.

Reply:
Before the change this would give me the error `fields.E009: max_length is too small to fit the longest value in choices (<count> characters).` They seem to add this check somewhere along the line here: https://code.djangoproject.com/ticket/30757

Reply:
Got it! Thanks for the link!